_haikus/Fall_Skies.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: haiku
+title: Fall Skies
+author: BrandonMGH
+---
+
+Pumpkins smile brightly<br>
```

Comment:
This line is one syllable over — I know it is less grammatical, but you can try
```
Pumpkins smile bright<br>
```
instead.